zstd_reader.go

```diff
@@ -0,0 +1,24 @@
+package zim
+
+import (
+	"io"
+
+	"github.com/klauspost/compress/zstd"
+)
+
+type ZstdReader struct {
+	*zstd.Decoder
+}
+
+func NewZstdReader(r io.Reader) (*ZstdReader, error) {
+	dec, err := zstd.NewReader(r)
+	if err != nil {
+		return nil, err
+	}
+	return &ZstdReader{dec}, nil
+}
+
+func (zr *ZstdReader) Close() error {
+	zr.Close()
+	return nil
```

Comment:
wy not `return  zr.Close()` ?

Reply:
well, `zr.Close()` doesn't return `error` itself. I just wanted to make it explicitly visible in the code that there won't ever be an error returned.

Reply:
Yep saw it while refactoring, thank you